enterprise/metrics/src/main/java/org/neo4j/metrics/source/DBMetrics.java

```diff
@@ -36,37 +37,58 @@
 
 import static com.codahale.metrics.MetricRegistry.name;
 
+@Documented( ".Database Metrics" )
 public class DBMetrics extends LifecycleAdapter
 {
     private static final String TRANSACTION_PREFIX = "neo4j.transaction";
-    private static final String TX_PEAK_CONCURRENT = name( TRANSACTION_PREFIX, "peak_concurrent" );
-    private static final String TX_STARTED = name( TRANSACTION_PREFIX, "started" );
-    private static final String TX_TERMINATED = name( TRANSACTION_PREFIX, "terminated" );
-    private static final String TX_ROLLBACKS = name( TRANSACTION_PREFIX, "rollbacks" );
-    private static final String TX_COMMITTED = name( TRANSACTION_PREFIX, "committed" );
-    private static final String TX_ACTIVE = name( TRANSACTION_PREFIX, "active" );
+    @Documented( "The highest peak of concurrent transactions ever seen on this machine" )
+    public static final String TX_PEAK_CONCURRENT = name( TRANSACTION_PREFIX, "peak_concurrent" );
+    @Documented( "The total number of started transactions" )
+    public static final String TX_STARTED = name( TRANSACTION_PREFIX, "started" );
+    @Documented( "The total number of terminated transactions" )
+    public static final String TX_TERMINATED = name( TRANSACTION_PREFIX, "terminated" );
+    @Documented( "The total number of rolled back transactions" )
+    public static final String TX_ROLLBACKS = name( TRANSACTION_PREFIX, "rollbacks" );
+    @Documented( "The total number of committed transactions" )
+    public static final String TX_COMMITTED = name( TRANSACTION_PREFIX, "committed" );
+    @Documented( "The number of currently active transactions" )
+    public static final String TX_ACTIVE = name( TRANSACTION_PREFIX, "active" );
 
     private static final String PAGE_CACHE_PREFIX = "neo4j.page_cache";
-    private static final String PC_EVICTION_EXCEPTIONS = name( PAGE_CACHE_PREFIX, "eviction_exceptions" );
-    private static final String PC_FLUSHES = name( PAGE_CACHE_PREFIX, "flushes" );
-    private static final String PC_UNPINS = name( PAGE_CACHE_PREFIX, "unpins" );
-    private static final String PC_PINS = name( PAGE_CACHE_PREFIX, "pins" );
-    private static final String PC_EVICTIONS = name( PAGE_CACHE_PREFIX, "evictions" );
-    private static final String PC_PAGE_FAULTS = name( PAGE_CACHE_PREFIX, "page_faults" );
+    @Documented( "The total number of exceptions seen during the eviction process in the page cache" )
+    public static final String PC_EVICTION_EXCEPTIONS = name( PAGE_CACHE_PREFIX, "eviction_exceptions" );
+    @Documented( "The total number of flushes executed by the page cache" )
+    public static final String PC_FLUSHES = name( PAGE_CACHE_PREFIX, "flushes" );
+    @Documented( "The total number of unpinned pages by the page cache" )
+    public static final String PC_UNPINS = name( PAGE_CACHE_PREFIX, "unpins" );
```

Comment:
This would be the number of page unpins. Not the count of how many pages are currently unpinned.